vue-app/src/components/TimeLeft.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <span :class="valueClass || 'value'">
+    {{ values[0] }}
+    <span :class="unitClass || 'unit'">
+      {{ units[0] }}
+    </span>
+    <span v-if="units[1].length > 0">
+      {{ units[1].length > 0 && values[1] }}
+      <span :class="unitClass || 'unit'">
+        {{ units[1] }}
+      </span>
+    </span>
+  </span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import Component from 'vue-class-component'
+import { DateTime } from 'luxon'
+import { Prop } from 'vue-property-decorator'
+
+import { getTimeLeft } from '@/utils/dates'
+import { TimeLeft } from '@/api/round'
+
+@Component
+export default class extends Vue {
+  @Prop() date!: DateTime
+  @Prop() valueClass!: string
+  @Prop() unitClass!: string
+
+  timeLeft: TimeLeft = getTimeLeft(this.date)
+  interval
+
+  mounted() {
+    this.interval = setInterval(() => {
+      this.timeLeft = getTimeLeft(this.date)
+    }, 1000)
+  }
+
+  beforeDestroy() {
+    clearInterval(this.interval)
+  }
+
+  updateTimeLeft(): TimeLeft {
```

Comment:
I think this method is not used.

Reply:
Good catch, wasn't needed in the end, just removed it 👍🏻